mediacloud/mediawords/tm/fetch_topic_tweets.py

```diff
@@ -0,0 +1,443 @@
+"""Use the Crimson Hexagon API to lookup tweets relevant to a topic, then fetch each of those tweets from twitter."""
+
+from abc import ABC, abstractmethod
+import datetime
+import re
+import time
+import tweepy
+
+from mediawords.db import DatabaseHandler
+import mediawords.util.json
```

Comment:
my understanding of python import is that 'import mediawords.util.json' imports less than 'from mediawords.util.json'.  That is to say that it makes the names in mediawords.util.json available in the current file but only when used with the mediawords.util.json prefix (mediawords.util.json.encode_json). 

I dislike importing the names of specific functions because then I have to go back to the top of the file to figure out which function is from where.  This is just a personal preference.